GiniVision/Classes/GiniPDFDocument.swift

```diff
@@ -56,6 +57,42 @@ final public class GiniPDFDocument: NSObject, GiniVisionDocument {
         }
     }
     
+    fileprivate func renderFirstPage(fromPdf pdf:CGPDFDocument) -> UIImage? {
+        var pdfImage:UIImage?
+        let pdfDoc = pdf
+        
+        if let pdfPage:CGPDFPage = pdfDoc.page(at: 1) {
+            var pageRect:CGRect = pdfPage.getBoxRect(.cropBox)
+            
+            // In case that the image was rotated 90 or 270, final rect should be rotated to portrait             
+            if pdfPage.rotationAngle == 90 || pdfPage.rotationAngle == 270 {
+                let tempWidth = pageRect.size.width
+                pageRect.size.width = pageRect.size.height
+                pageRect.size.height = tempWidth
+            }
+            
+            // Create context
```

Comment:
Useless comment, IMHO

Reply:
I could look like copied from SO, but actually they were checks for me since I was struggling with an issue related to image orientation.
For me they were useless at all, but if you want they could be removed.